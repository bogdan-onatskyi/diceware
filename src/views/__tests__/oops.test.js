import React from 'react';

import Oops from "../oops";

describe("Oops component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <Oops/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it("should not have any props", () => {
        expect(Oops.propTypes).toBe(undefined);
    });
});