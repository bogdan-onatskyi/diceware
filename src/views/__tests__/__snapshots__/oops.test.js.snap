// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Oops component should match saved snapshot 1`] = `
<div
  className="app"
>
  <Header
    text="Не реализовано еще..."
  />
  <Menu />
  <div
    className="app__content app__content--oops"
  >
    <div
      className="app__content app__content--oops-box"
    >
      <p>
        Ой, то что Вы запросили, еще не реализовано, но я над этим работаю...
      </p>
    </div>
  </div>
  <Footer />
</div>
`;
